lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -156,6 +174,40 @@ def add_echeck(params, echeck, options)
         params[:tele_check] = tele_check
       end
 
+      def use_token(params, payment_method, options)
```

Comment:
I think `add_` is more idiomatic in AM than `use_` which is less informative.

Reply:
My thought was the `add` in `add_payment_method` would suffice and that the `add` here was confusing in the context of adding a tokenized card. Happy to switch it back if `use` isn't working.

Reply:
👍 IME, any method that directly adds values to the hash that will become the request uses `add_`.